lascli/parser/datasets.py

```diff
@@ -96,6 +99,67 @@ def parse_csv(csv_path, delimiter=','):
     return documents
 
 
+def read_json(path, encoding):
+    try:
+        return json.loads(path.read_text(encoding))
+    except (UnicodeDecodeError, json.JSONDecodeError):
+        pass
+
+
+def read_yaml(path, encoding):
+    try:
+        return yaml.safe_load(path.read_text(encoding))
+    except (UnicodeDecodeError, ParserError):
+        pass
+
+
+def read_ground_truth(path, encoding):
+    return read_json(path, encoding) or read_yaml(path, encoding)
```

Comment:
Shouldn't we throw an error here if both are None?

Reply:
It will be caught in `_create_documents_worker` and written to error file as far as I can tell